src/ChargeDriftModels/ADLChargeDriftModel/TemperatureModels/PowerLawModel.jl

```diff
@@ -1,57 +1,88 @@
+* `theta_h100::T`: Characteristic temperature $\theta$ for holes along the <100> axis.
+
+"""
+
+mutable struct PowerLawModel{T <: SSDFloat} <: AbstractTemperatureModel{T}
+    reference_temperature::T
+    p_e100::T
+    theta_e100::T
+    p_h100::T
+    theta_h100::T
 end
 
 function PowerLawModel(config_file::AbstractDict; T::Type{<:AbstractFloat} = Float32)::PowerLawModel
     return PowerLawModel{T}(config_file::AbstractDict)
 end
 
-function PowerLawModel{T}(config_file::AbstractDict; temperature::Union{Missing, T} = missing)::PowerLawModel where {T <: SSDFloat}
+function PowerLawModel{T}(config_file::AbstractDict)::PowerLawModel where {T <: SSDFloat}
     config_file["temperature_dependence"]["model"] != "PowerLaw" ? error() : nothing
-    ismissing(temperature) ? temperature = config_file["temperature_dependence"]["temperature"] : nothing
     m = PowerLawModel{T}(
-        temperature, #temperature = config_file["temperature_dependence"]["temperature"]
-        config_file["drift"]["velocity"]["temperature"],
-        config_file["temperature_dependence"]["parameters"]["e100"]["p1"],
-        config_file["temperature_dependence"]["parameters"]["e111"]["p1"],
-        config_file["temperature_dependence"]["parameters"]["h100"]["p1"],
-        config_file["temperature_dependence"]["parameters"]["h111"]["p1"]
+        _parse_value(T, config_file["temperature_dependence"]["reference_temperature"], u"K"),
+        _parse_value(T, config_file["temperature_dependence"]["parameters"]["e100"]["p"], NoUnits),
+        _parse_value(T, config_file["temperature_dependence"]["parameters"]["e100"]["theta"], u"K"),
+        _parse_value(T, config_file["temperature_dependence"]["parameters"]["h100"]["p"], NoUnits),
+        _parse_value(T, config_file["temperature_dependence"]["parameters"]["h100"]["theta"], u"K")
     )
 end
 
-function scale_to_given_temperature(m::PowerLawModel{T})::NTuple{4,T} where {T <: SSDFloat}
-
-    function f(p1::T = 0.0)::T where {T <: SSDFloat}
-        return (m.reftemperature/m.temperature)^1.5
-    end
-
-    scale_e100 = f(m.p1e100)
-    scale_e111 = f(m.p1e111)
-    scale_h100 = f(m.p1h100)
-    scale_h111 = f(m.p1h111)
-
-    return scale_e100, scale_e111, scale_h100, scale_h111
-end
-
-print(io::IO, tm::PowerLawModel{T}) where {T <: SSDFloat} = print(io, "PowerLawModel{$T}")
-function println(io::IO, tm::PowerLawModel{T}) where {T <: SSDFloat}
-    println("\n________PowerLawModel________")
-    println("Fit function: p1 * T^(3/2)\n")
-    println("---Temperature settings---")
-    println("Crystal temperature:   \t $(tm.temperature)")
-    println("Reference temperature: \t $(tm.reftemperature)\n")
-
-    println("---Fitting parameters---")
-    println("   \te100      \te111      \th100      \th111")
-    println("p1 \t$(tm.p1e100)   \t$(tm.p1e111)   \t$(tm.p1h100)   \t$(tm.p1h111)")
-end
+function scale_to_temperature(cdm::ADL2016ChargeDriftModel{T,M,N,PowerLawModel{T}}, Temp::RealQuantity) where {T,M,N}
```

Comment:
Can we add the same temperature dependence for the standard `ADLChargeDriftModel`, and maybe even the other ones (`ConstantMobilityChargeDriftModel` could just be scaled with the power law, and the saturation drift velocity T-dependence is ignored).